### DISCUSSION TITLE
Use fzf Docker image

### DISCUSSION THREAD

**Discussion by electriquo** [junegunn/fzf#3120] [Q&A] [answered]
The project holds a [Dockerfile](https://github.com/junegunn/fzf/blob/master/Dockerfile). I tried my best to search around Docker Hub and other Docker registries but couldn't find an official published image of the project.

If there is an official Docker image release, please point me to it.
If there is no official Docker image release, is there any intention to publish it?

**Comment by junegunn** [comment, marked as answer]
No, it is only for testing. fzf is a command-line utility and it doesn't make much sense to distribute it as a Docker image.

**Reply by electriquo** [reply]
You could claim the same thing on every command line utility, such as curl. Yet, some command line utilities are distributed as a Docker image since it can be used in the both the Docker realm and outside of it.

For example, I would love to use fzf for searching for files and exploring the logs content within a K8s cluster.

Kindly reconsider to distribute fzf as a Docker image.

**Reply by junegunn** [reply]
What prevents you from using fzf on a Docker container? I really don't see the point.

**Reply by electriquo** [reply]
Nothing prevents me from using fzf in a Docker container. I thought of utilizing an official fzf Docker image and not to maintain one outside of fzf project.

**Reply by junegunn** [reply]
fzf by itself is pretty much useless as it is just a text filter. You would need other tools to do something useful with the output of fzf. (Like, what good is a Docker image just for `grep`?)

**Reply by electriquo** [reply]
busybox, ubuntu, alpine, etc.
